submodules/SSignalKit/SwiftSignalKit/Source/Signal_Mapping.swift

```diff
@@ -128,3 +128,21 @@ public func distinctUntilChanged<T, E>(isEqual: @escaping (T, T) -> Bool) -> (_
         }
     }
 }
+
+///Converts error  to value.
```

Comment:
This extension based on `asDriver(onErrorJustReturn:` from [RxSwift](https://github.com/ReactiveX/RxSwift/blob/main/RxCocoa/Traits/Driver/ObservableConvertibleType+Driver.swift)